test/prepare_disk.sh

```diff
@@ -379,7 +379,7 @@ done
 shift "$((OPTIND - 1))"
 
 if [ "${#}" -ne 1 ]; then
-    echo "Missing argument <file.img>."
+    echo "Missing argument <toolbox image file>."
```

Comment:
sneaked this change unrelated to this changeset in here :) and completly unessasery as you added this in the first commit of this changeset.